src/backend/joanie/demo/management/commands/create_dev_demo.py

```diff
@@ -60,23 +62,124 @@ def create_course(self, user, organization, batch_size=1, with_course_runs=False
+                    is_listed=True,
+                    state=CourseState.ONGOING_OPEN,
+                    languages=self.get_random_languages(),
+                    resource_link=OPENEDX_COURSE_RUN_URI.format(
+                        course=target_course.code, course_run="{course.title}_run1"
+                    ),
+                )
+                factories.CourseRunFactory(
+                    course=target_course,
+                    is_listed=True,
+                    state=CourseState.ONGOING_OPEN,
+                    languages=self.get_random_languages(),
+                    resource_link=OPENEDX_COURSE_RUN_URI.format(
+                        course=target_course.code, course_run="{course.title}_run2"
+                    ),
+                )
+            return product
+        return [
+            self.create_product_credential(user, organization)
+            for i in range(batch_size)
+        ]
+
+    def create_product_certificate(self, user, organization, batch_size=1):
+        """Create batch or products certificate for given user and organization."""
+        if batch_size == 1:
+            course = factories.CourseFactory(
+                organizations=[organization],
+                users=[[user, enums.OWNER]],
+            )
+            factories.CourseRunFactory(
+                course=course,
+                is_listed=True,
+                state=CourseState.ONGOING_OPEN,
+                languages=self.get_random_languages(),
+                resource_link=OPENEDX_COURSE_RUN_URI.format(
+                    course=course.code, course_run="{course.title}_run1"
+                ),
+            )
+            factories.CourseRunFactory(
+                course=course,
+                is_listed=True,
+                state=CourseState.ONGOING_OPEN,
+                languages=self.get_random_languages(),
+                resource_link=OPENEDX_COURSE_RUN_URI.format(
+                    course=course.code, course_run="{course.title}_run2"
+                ),
+            )
+            product = factories.ProductFactory(
+                type=enums.PRODUCT_TYPE_CERTIFICATE, courses=[course]
+            )
+            return product
+        return [
+            self.create_product_certificate(user, organization)
+            for i in range(batch_size)
+        ]
+
+    def create_product_certificate_enrollment(self, user, organization):
+        product = self.create_product_certificate(user, organization)
+        course = product.courses.first()
+        return factories.EnrollmentFactory(
+            user=user,
+            course_run=course.course_runs.first(),
+            is_active=True,
+            state=enums.ENROLLMENT_STATE_SET,
         )
-        return factories.ProductFactory(
-            courses=[course],
-            target_courses=target_courses,
-            type=enums.PRODUCT_TYPE_CREDENTIAL,
+
+    def create_product_certificate_order(self, user, organization):
```

Comment:
`create_product_certificate_purchased` is more explicit as name nope ? 
Then the function above could me renamed `create_product_certificate_with_related_enrollment`